security/abac/policy-el/src/main/java/io/helidon/security/abac/policy/el/JavaxElPolicyExecutor.java

```diff
@@ -116,11 +116,8 @@ public void executePolicy(String policyStatement, Errors.Collector collector, Pr
                 collector.fatal(this, "Policy statement \"" + policyStatement + "\" evaluated to false");
             }
         } catch (Exception e) {
-            collector.fatal(this,
-                            "Policy statement \"" + policyStatement + "\" evaluated to an exception " + e.getClass()
-                                    .getName() + " with message: " + e.getMessage());
-
             LOGGER.log(Level.FINEST, e, () -> "Statement " + policyStatement + " evaluation failed");
+            throw new SecurityException("Policy statement \"" + policyStatement + "\" evaluated to an exception", e);
```

Comment:
Is logging still needed here?

Reply:
The security exception is propagated outside of security (and outside of Helidon code), so yes, I think it is reasonable to log it (that is why it is in `FINEST` level)